unstructured/ingest/__init__.py

```diff
@@ -1 +1,7 @@
 from __future__ import annotations
+
+import warnings
```

Comment:
Should we use the logger from `unstructurued.logger` for this?

Reply:
From what I read, this is the recommended way to mark a warning.